redirects/linux.nginx

```diff
@@ -0,0 +1,344 @@
+     return 301 https://min.io/minio/docs/linux/administration/monitoring.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/bucket-notifications/publish-events-to-amqp.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring/publish-events-to-amqp.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/bucket-notifications/publish-events-to-mqtt.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring/publish-events-to-mqtt.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/bucket-notifications/publish-events-to-nats.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring/publish-events-to-nats.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/bucket-notifications/publish-events-to-nsq.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring/publish-events-to-nsq.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/bucket-notifications/publish-events-to-elasticsearch.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring/publish-events-to-elasticsearch.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/bucket-notifications/publish-events-to-kafka.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring/publish-events-to-kafka.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/bucket-notifications/publish-events-to-mysql.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring/publish-events-to-mysql.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/bucket-notifications/publish-events-to-postgresql.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring/publish-events-to-postgresql.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/bucket-notifications/publish-events-to-redis.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring/publish-events-to-redis.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/monitoring/bucket-notifications/publish-events-to-webhook.html {
+     return 301 https://min.io/minio/docs/linux/administration/monitoring/publish-events-to-webhook.html?ref=docs-redirect;
+}
+
+
+
+
+# Reference Redirects from NuDocs 1 to NuDocs 2
+# =============================================
+
+
+location /minio/baremetal/reference/minio-mc.html {
+     return 301 https://min.io/minio/docs/linux/reference/minio-mc.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/reference/minio-mc/ {
+     rewrite /minio/baremetal/reference/minio-mc/(.*) https://min.io/minio/docs/linux/reference/minio-mc/$1 permanent;
+}
+
+location /minio/baremetal/reference/minio-mc-admin.html {
+     return 301 https://min.io/minio/docs/linux/reference/minio-mc.html?ref=docs-redirect;
+}
+
+location /minio/baremetal/reference/minio-mc/ {
```

Comment:
`/minio/baremetal/reference/minio-mc-admin/`

Otherwise, this is a repeat of line 291.